thunder/benchmarks/benchmark_scripts.py

```diff
@@ -0,0 +1,117 @@
+import json
+import os
+import subprocess
+
+from typing import Any
+from datetime import datetime
+from functools import partial
+
+from absl import flags
+from absl import logging
+from absl.testing import absltest, parameterized
+
+flags.DEFINE_string("benchmark_output_dir", default="./benchmark_results", help="Output directory for benchmark JSONs")
```

Comment:
```suggestion
flags.DEFINE_string("output_dir", default="./benchmark_results", help="Output directory for benchmark JSONs")
```
As it is passed to the benchmark script...